tests/Unit/Models/VoucherStateModelTest.php

```diff
@@ -17,20 +18,22 @@ class VoucherStateModelTest extends TestCase
 {
     use DatabaseMigrations, DatabaseTransactions;
 
-    protected $user;
+    protected $marketUser;
+    protected $adminUser;
 
     public function setUp()
     {
         parent::setUp();
-        $this->user = factory(User::class)->create();
+        $this->marketUser = factory(User::class)->create();
+        $this->adminUser = factory(AdminUser::class)->create();
     }
 
-
+    /** @test */
```

Comment:
I think either use this comment or begin method name with test. Confusing to do both.